bucket/jdtls.json

```
@@ -0,0 +1,29 @@
+{
+    "version": "1.10.0-202204081744",
+    "description": "Java language server",
+    "homepage": "https://projects.eclipse.org/projects/eclipse.jdt.ls",
+    "license": "EPL v2.0",
+    "suggest": {
+        "JDK": "java/openjdk",
+        "Python3": "python"
+    },
+    "url": "https://download.eclipse.org/jdtls/snapshots/jdt-language-server-1.10.0-202204081744.tar.gz",
+    "hash": "da4c832d89e1e66932eb809c5760043fe1abdbefc1a4492a8022edd354c873c5",
+    "installer": {
+        "script": "scoop shim add jdtls $scoopdir\\apps\\python\\current\\python.exe $dir\\bin\\jdtls"
+    },
+    "uninstaller": {
+        "script": "scoop shim rm jdtls"
+    },
```

```suggestion
    "bin": "bin\\jdtls.py",
```

You can directly do this

---

Yeah sorry I should had explained the reasoning behind this!

Currently the "bin" option in scoop generates a NON-exe shim if used like the above suggestion; this initially might seem ok, but it cannot be used with programs that need native executables to use it and not powershell scripts.

Such an example is the `neovim` language server client and any other program that relies on `libuv`'s `spawn` method (see here: https://github.com/libuv/libuv/blob/2a1b880f5439e074ef4d5c556f626b1044fb6781/src/win/process.c#L297)

The `depend` option seems a good idea, I will modify the manifest to depend on `python`.

Finally as a side note, the current PR will allow scoop + neovim users to setup painlessly their java development environment when this PR lands too!: https://github.com/neovim/nvim-lspconfig/pull/1833